src/clib/CMakeLists.txt

```diff
@@ -1,15 +1,16 @@
 cmake_minimum_required (VERSION 2.8.12)
 include (CheckFunctionExists)
-project (PIOC C)
+project (PIOC C CXX)
```

Comment:
I really, really doubt we want to introduce C++ into the mix here. I have not yet encountered the C++ code that has been added, but my initial thought is that it should simply be translated as C.

The introduction of a new language to the build system adds a cost in support and portability. What are we getting from this that could not be done in C or Fortran?